### PR TITLE
CM-80 IVDCMModule integration/implementation

### PR DESCRIPTION
**Pull request by VladSemenyuk**

### CODE REVIEW

**Comment by VladSemenyuk** [comment]
@bonewell

File: src/components/ivdcm_module/include/ivdcm_module/ivdcm.h [hunk 1]
```diff
@@ -37,14 +37,12 @@
 #include "functional_module/generic_module.h"
 #include "json/value.h"
 #include "utils/macro.h"
-#include "utils/threads/message_loop_thread.h"
 
 namespace ivdcm_module {
 
 typedef Json::Value MessageFromMobile;
```
Comment (bonewell) [inline src/components/ivdcm_module/include/ivdcm_module/ivdcm.h]: I think this typedef can be removed.

**Comment by bonewell** [comment]
Reviewed.

**Comment by bonewell** [comment]
Reviewed.

**Comment by DKlimenko** [comment]
reviewed

**Pull request merged**